docs/cni/kubernetes/KubernetesIntegration.md

```diff
@@ -41,37 +45,114 @@ sudo chmod +x calicoctl
+> Replace <ETCD_IP>:<ETCD_PORT> with your etcd configuration.
 
 ## Configuring Kubernetes
-#### Configuring the Kubelet 
-Once the Calico network plugin has been installed, and the `calico/node` container started, you will need to configure the Kubelet to use the Calico network plugin when starting pods. 
+### Configuring the Kubelet
+The Kubelet needs to be configured to use the Calico network plugin when starting pods.
+
+The `kubelet` can be configured to use Calico by starting it with the following options
+- `--network-plugin=cni`
+- `--network-plugin-dir=/etc/cni/net.d`
 
-The `kubelet` can be configured to use Calico by starting it with the `--network-plugin=calico` option.
+See the [`kubelet` documentation](http://kubernetes.io/v1.1/docs/admin/kubelet.html)
+for more details.
 
-On CoreOS machines, the Calico network plugin is installed to the `/etc/kubelet-plugins` directory, and so an additional flag is necessary to inform the `kubelet` of the location of the plugin.  On CoreOS, include the `--network-plugin-dir=/etc/kubelet-plugins` option when starting the `kubelet`.
-> Note: the `--network-plugin-dir=` option is only available in Kubernetes >= v1.1.0
+#### Example systemd unit file (kubelet.service)
+```
+[Unit]
+Description=Kubernetes Kubelet
+Documentation=https://github.com/kubernetes/kubernetes
+After=calico-node.service
+Requires=calico-node.service
+
+[Service]
+ExecStartPre=/usr/bin/wget -N -P /opt/bin https://storage.googleapis.com/kubernetes-release/release/v1.1.3/bin/linux/amd64/kubelet
+ExecStartPre=/usr/bin/chmod +x /opt/bin/kubelet
+ExecStartPre=/usr/bin/wget -N -P /opt/cni/bin https://github.com/projectcalico/calico-cni/releases/download/v0.2.0/calico
+ExecStartPre=/usr/bin/chmod +x /opt/cni/bin/calico
+ExecStart=/opt/bin/kubelet \
+--address=0.0.0.0 \
+--allow-privileged=true \
+--cluster-dns=10.100.0.10 \
+--cluster-domain=cluster.local \
+--config=/etc/kubernetes/manifests \
+--hostname-override=$private_ipv4 \
+--api-servers=http://<API SERVER IP>:8080 \
+--network-plugin-dir=/etc/cni/net.d \
+--network-plugin=cni \
+--logtostderr=true
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+```
 
-#### Configuring the Kube-Proxy
-In order to use Calico policy with Kubernetes, the `kube-proxy` component must be configured to leave the source address of service bound traffic intact.  This feature is first officially supported in Kubernetes v1.1.0.
+This unit file ensures that the `kubelet` binary and the `calico` plugin are present.
+
+### Configuring the Kube-Proxy
+In order to use Calico policy with Kubernetes, the `kube-proxy` component must
+be configured to leave the source address of service bound traffic intact.
+This feature is first officially supported in Kubernetes v1.1.0.
 
 There are two ways to enable this behavior.
 - Option 1: Start the `kube-proxy` with the `--proxy-mode=iptables` option.
```

Comment:
I believe that as of v1.1.3, this is the default behavior of the kube-proxy, and so we don't need to use a special flag.